cmd/daytona/config/const.go

```diff
@@ -95,6 +98,8 @@ func GetScopesFromGitProvider(providerId string) string {
 		return "read:organization,write:repository,read:user"
 	case "gitness":
 		return "/"
+	case "azure-devops":
+		return "vso.code_write, vso.profile, vso.project"
```

Comment:
Can you explain these scope names: `vso.code_write, vso.profile, vso.project`

Why were they listed instead of the labels appearing in the UI? I can see the "help" link you used also has screenshots from the UI

![image](https://github.com/daytonaio/daytona/assets/25279767/3407ecf6-1089-4c59-9a23-67a8167e0220)